frontend/src/components/StyledComponents.tsx

```diff
@@ -0,0 +1,17 @@
+import { Dialog } from '@equinor/eds-core-react'
```

Comment:
Unsure about having this file directly under components. How about creating a Styles folder and moving it in there? 

Also, why is it only these two styles that are moved out?

Reply:
Because they are the most common ones and we need further refactoring to be able to start re-using the same styles. But I think in the future that is what we should want to do so the file already exists.

Reply:
I also created the folder like you suggested